cypress/e2e/ui.cy.js

```diff
@@ -0,0 +1,70 @@
+/*
+  Tests for different buttons / paths the user may take. 
+  Mostly to prove some user stories are complete
+*/
+
+describe('Button Tests', () => {
+    it('Buttons for Localities, Species, References & Time Units are visible', () => {
+      cy.visit('/')
+      cy.contains('Welcome').should('be.visible')
+      cy.contains('Front Page').should('be.visible')
+      cy.contains('Localities').should('be.visible')
+      cy.contains('Species').should('be.visible')
+      cy.contains('References').should('be.visible')
+      cy.contains('Time Units').should('be.visible')
```

Comment:
This should probably include the cross-search page also

Reply:
added these, also for map